samples/dashboards/demos/custom-html-component/demo.js

```diff
@@ -0,0 +1,30 @@
+const customHTML = document.getElementById('custom-html').outerHTML,
+    AST = Highcharts.AST;
+
+Dashboards.board('container', {
+    gui: {
+        layouts: [{
+            id: 'layout-1',
+            rows: [{
+                cells: [{
+                    id: 'dashboard-col-0'
+                }, {
+                    id: 'dashboard-col-1'
+                }]
+            }]
+        }]
+    },
+    components: [{
+        type: 'HTML',
+        cell: 'dashboard-col-0',
+        elements: new AST(`${customHTML}`).nodes
+    }, {
```

Comment:
It's not really custom component. Why don't we create a **real** custom component, where instead of adding `element` you add new `type: 'HTMLString'` and 
```html: myHtml```